modules/ensemble/lib/widget/webview/web/webviewstate.dart

```diff
@@ -29,18 +30,18 @@ class WebViewState extends EWidgetState<EnsembleWebView> {
     _iframeElement.style.width = '100%';
     _iframeElement.style.height = '100%';
 
-    _iframeElement.src = widget.controller.url;
+    _iframeElement.src = widget.controller.url ?? '';
     _iframeElement.style.border = 'none';
 
     // ignore: undefined_prefixed_name
     ui.platformViewRegistry.registerViewFactory(
-      'iframeElement',
+      viewId,
       (int viewId) => _iframeElement,
     );
 
     return HtmlElementView(
       key: UniqueKey(),
-      viewType: 'iframeElement',
+      viewType: viewId,
     );
```

Comment:
we should add a dispose method here and dispose off the htmlView and iframeElement. That should get rid of the memory leak